modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -3,100 +3,102 @@
-| `EvictionPolicy EvictionPolicyType` | The type of the eviction to use for the bucket, defaults to `valueOnly`. | true (note: changing will cause the bucket to restart causing temporary inaccessibility)
-| `MaxTTL time.Duration` | The default maximum time-to-live to apply to documents in the bucket. (note: This option is only available for Couchbase and Ephemeral buckets in Couchbase Enterprise Edition.) | true
-| `CompressionMode CompressionMode` | The compression mode to apply to documents in the bucket. (note: This option is only available for Couchbase and Ephemeral buckets in Couchbase Enterprise Edition.) | true
-| `ConflictResolutionType ConflictResolutionType` | The conflict resolution type to apply to conflicts on the bucket, defaults to `seqno` | false
+| `name string` | The name of the bucket, required for creation. | false
+| `flushEnabled boolean` | Enables flushing to be performed on this bucket (see the <<java-flushing>> section below). | true
+| `replicaIndexes boolean` | Whether or not to replicate indexes. | false
+| `ramQuotaMB uint64` | How much memory should each node use for the bucket, required for creation. | true
+| `numReplicas` int | The number of replicas to use for the bucket. | true
+| `bucketType BucketType` | The type of the bucket, required for creation. | false
+| `evictionPolicy EvictionPolicyType` | The type of the eviction to use for the bucket, defaults to `VALUE_ONLY`. | true (note: changing will cause the bucket to restart causing temporary inaccessibility)
+| `maxExpiry time.Duration` | The default maximum time-to-live to apply to documents in the bucket. (note: This option is only available for Couchbase and Ephemeral buckets in Couchbase Enterprise Edition.) | true
+| `compressionMode CompressionMode` | The compression mode to apply to documents in the bucket. (note: This option is only available for Couchbase and Ephemeral buckets in Couchbase Enterprise Edition.) | true
+| `conflictResolutionType ConflictResolutionType` | The conflict resolution type to apply to conflicts on the bucket, defaults to `SEQUENCE_NUMBER` | false
 |====
 
 The following example creates a "hello" bucket:
 
-[source,golang,indent=0]
+[source,java]
 ----
-include::devguide:example$go/provisioning-resources-buckets.go[tag=createBucket]
+include::example$ProvisioningResourcesBuckets.java[tag=createBucket,indent=0]
 ----
 
 We can now get this bucket and update it to enable Flush:
 
-[source,golang,indent=0]
+[source,java]
 ----
-include::devguide:example$go/provisioning-resources-buckets.go[tag=updateBucket]
+include::example$ProvisioningResourcesBuckets.java[tag=updateBucket,indent=0]
 ----
 
 Once you no longer need to use the bucket, you can remove it:
 
-[source,golang,indent=0]
+[source,java]
 ----
-include::devguide:example$go/provisioning-resources-buckets.go[tag=removeBucket]
+include::example$ProvisioningResourcesBuckets.java[tag=removeBucket,indent=0]
 ----
-////
 
-[#go-flushing]
+[#java-flushing]
```

Comment:
Just a curiosity - why are these IDs tagged with language? Will we have multiple SDKs included on same page in some cases? (e.g. this is for the dev-reorg guides?)